tests/common/transactions.py

```diff
@@ -0,0 +1,24 @@
+from eth_account import Account
+from eth_utils import to_hex
+import rlp
+from eth_account._utils.legacy_transactions import Transaction
+
+
+def encode_transaction_data(data: list) -> str:
```

Comment:
This is the custom way of encoding the transactions in the backend integration tests that produces the same encoding than the frontend code